### PR TITLE
[WIP] Add: gps-exporter service

### PR DESCRIPTION
**Pull request by abalmos**

### CODE REVIEW

File: services/gps-exporter/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+use ansi_term::Colour::{Blue, Green, Red, White, Yellow};
+
+#[derive(sqlx::FromRow)]
+struct GPS {
+    time: DateTime<Utc>,
+    lat: f64,
+    lng: f64,
+}
+
+// TOOD: remove --hourly and --daily and make -l take <num|hourly|daily> so that
```
Comment (aaron97neu) [inline services/gps-exporter/src/main.rs:24]: Are you planning on implementing this in this version?

Comment (aaron97neu) [inline services/gps-exporter/src/main.rs:24]: Are you planning on implementing this in this version?

File: services/gps-exporter/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+        matches
+            .value_of("out_dir")
+            .with_context(|| "Output directory is required.")?,
+    );
+
+    if !out_dir.exists() {
+        std::fs::create_dir(out_dir).with_context(|| "Can not create output directory")?;
+    }
+
+    // TODO: --append implantation is a bit ... nasty?
```
Comment (aaron97neu) [inline services/gps-exporter/src/main.rs:162]:
```suggestion
    // TODO: --append implementation is a bit ... nasty?
```

File: services/gps-exporter/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+fn is_u64(num: String) -> Result<(), String> {
+    match num.parse::<u64>() {
+        Ok(_) => Ok(()),
+        Err(_) => Err(format!("Must be a positive integer")),
+    }
+}
+
+/*
+#[derive(Debug)]
```
Comment (aaron97neu) [inline services/gps-exporter/src/main.rs:295]: Wouldn't mind a comment here explaining that the shapefile lib is broken and a link that will let us know the status of the fix

**Review by aaron97neu — APPROVED** [review]
Few small things if we can. None are serious so I don't mind merging without

**Comment by aaron97neu** [comment]
Also add your ownership of this container to the end of the CODEOWNERS file [here](https://github.com/oats-center/isoblue-avena/blob/feature/gps-exporter/.github/CODEOWNERS) in the form `services/gps-exporter @abalmos`

**Review by facastiblancor — APPROVED** [review]

**Pull request merged**